jenkinsfiles/ginkgo-gke.Jenkinsfile

```diff
@@ -151,6 +164,7 @@ pipeline {
     post {
         always {
             sh 'cd ${TESTDIR}/gke; ./release-cluster.sh || true'
+            sh 'cd ${TESTDIR}; ./clean-local-registry-tag.sh $(./print-node-ip.sh) ${TAG}'
```

Comment:
Does this also need a `|| true` at the end?

Reply:
Will add